.pre-commit-hooks.yaml

```
@@ -1,3 +1,23 @@
+- id: flake8-ros
+  name: flake8 ros
+  description: Apply flake8 with the ROS 2 settings.
+  entry: flake8
+  language: python
+  types: [python]
+  # require_serial: true
```

I'm not sure if we should set `require_serial`.
https://github.com/PyCQA/flake8/blob/463d9b1ce9ee44665f2c6825edd7010ef5dff760/.pre-commit-hooks.yaml#L7

@KeisukeShima Are you familiar with this?

```suggestion
  require_serial: true
```

---

> (optional: default false) if true this hook will execute using a single process instead of in parallel. new in 1.13.0.

If flake8 can run on multiple python files in parallel, you can set this to true.

---

> you can set this to true.

Is this a mistake of `you can set this to false`? :thinking:

---

Ah, yes. I meant to say, "you can set this to false".

---

Ah, it should be true.
https://github.com/PyCQA/flake8/commit/535775e646258428fb922ef6f15e46d268cd4789
`flake8 internally implements multiprocessing, disabling pre-commit's improves performance`